<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RSI1505" ignored="0" name="br.com.rsinet.hub_TDD" tests="6" failures="1" timestamp="2020-01-27T10:56:13 BRT" time="144.71" errors="0">
  <testcase name="testaBuscaLupaInvalido" time="5.374" classname="br.com.rsinet.hub_TDD.testes.TestBuscaLupa"/>
  <testcase name="testaBuscaLupaValido" time="5.643" classname="br.com.rsinet.hub_TDD.testes.TestBuscaLupa"/>
  <testcase name="testaBuscaCliqueInvalido" time="9.928" classname="br.com.rsinet.hub_TDD.testes.TesteBuscaClique"/>
  <testcase name="testaBuscaCliqueValido" time="12.154" classname="br.com.rsinet.hub_TDD.testes.TesteBuscaClique"/>
  <testcase name="testeCadastroValido" time="21.77" classname="br.com.rsinet.hub_TDD.testes.TesteCadastro">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at br.com.rsinet.hub_TDD.testes.TesteCadastro.testeCadastroValido(TesteCadastro.java:91)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testeCadastroValido -->
  <testcase name="testeCadastroInvalido" time="23.827" classname="br.com.rsinet.hub_TDD.testes.TesteCadastro"/>
</testsuite> <!-- br.com.rsinet.hub_TDD -->
